Blog: Jumping Rivers' Litmusverse: Another Success for Open-Source Validation

jcabercrombie opened the pull request:
Publishing the _Jumping Rivers' Litmusverse: Another Success for Open-Source Validation_ blog to our website. 

Pending final review to run by more R Validation Hub members and JR members if they wish to read beforehand.

Commit:
initial commit for JR blogpost

Commit:
add litmus verse image

Commit:
add subheaders, bolding/italicizing, and image for better visual appeal

Commit:
confine litmusverse image pixels to fit page and rerender page

dgkf commented on static/img/banners/jumping_rivers_litmusverse.png:
This might be a bit too much brand-fusion. I'm not sure I'm comfortable putting our logo here since litmus is far more of a JR product than our product at this point. Let's check with JR.

jcabercrombie replied:
Ah, I see what you are saying. This was just a product of our template we use for all blog posts and I just added the package hex. I can easily remove our logo from it!

jcabercrombie replied:
FYI @dgkf - This has been updated now!

dgkf commented:
Hi @csgillespie - we're hoping to put out a post that recognizes the JR accomplishment of announcing `{litmus}` and draw attention to the shared internals, goals and investment in the ecosystem.

We want to make sure you've had a first look at the post and an opportunity to correct anything before it goes live. Please take a look!

Commit:
update banner image to exclude R Validation Hub from logo

Commit:
merge from master; riskassessment blog

Merge branch 'master' into jcabercrombie_jr_blog
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

Commit:
Adding litmus dashboard

Commit:
rerender for litmus dashboard

Commit:
add litmus dashboard image for blogpost

Commit:
fix category and tags

Commit:
re-knit after fixing category and tags

Commit:
merge from master

Merge branch 'master' into jcabercrombie_jr_blog
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

Commit:
merge with master

Merge branch 'master' into jcabercrombie_jr_blog
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

Commit:
remove HTML given new repo structure and necessary files

Commit:
change from rmd to md

Commit:
remove width text now that file is md
